src/pages/workspace/accounting/netsuite/export/NetSuiteExportExpensesJournalPostingPreferenceSelectPage.tsx

```diff
@@ -0,0 +1,74 @@
+import {useRoute} from '@react-navigation/native';
```

Comment:
Comparing this and the previous page except for a few types, the whole page is same. Isn't there a way to just merge these into one and DRY the code?